setup.py

```diff
@@ -13,7 +13,7 @@
   download_url = 'https://github.com/bodyshopbidsdotcom/githubgateway/tarball/%s' % current_version,
   keywords = ['api', 'gateway', 'http', 'REST'],
   install_requires = [
-    'basegateway==0.12'
+    'basegateway==0.13'
```

Comment:
It was a bad idea on my part to lock it to a specific version, let's do any `0` version